redis/src/test/scala/zio/redis/OutputSpec.scala

```diff
@@ -498,7 +498,182 @@ object OutputSpec extends BaseSpec {
             )
           )
           Task(XReadOutput.unsafeDecode(input)).either.map(assert(_)(isLeft(isSubtype[ProtocolError](anything))))
-        }
+        },
+        suite("XInfo stream commands output")(
+          testM("stream extract valid value") {
+            val input = RespValue.array(
```

Comment:
I suggest that you rename this value to `resp`. The same goes for other cases in output spec.